packages/hardhat/contracts/FundingVault.sol

```diff
@@ -56,13 +57,34 @@ contract FundingVault{
     uint256 private blockLimit;
     uint256 private minFundingAmount; //The minimum amount of ETH required in the contract to enable withdrawal.
     uint256 private exchangeRate; //The exchange rate of ERG per token
+    address private projectOwner;
     address private withdrawlAddress;
```

Comment:
typo here.

Reply:
Adding projectOwner address also causes solidity function stack too deep, try removing local variables error...
currently considering withdrawl address as owner in the frontend. Please suggest possible changes..

Reply:
To be clear, is the owner msg.sender in fundingVaultFactory? I am trying to remove stack too deep error.

Reply:
@DengreSarthak Please don't rush also it's not a hotfix and It will be good to have Brunu suggestion here because i already have number of solution in my mind... I will recommend you tryout frontend first